content/guides/use-case/jupyter/run-jupyter.md

```diff
@@ -0,0 +1,175 @@
+---
+description: Learn how to run a personal Jupyter Server with the JupyterLab using Docker.
+keywords: jupyter, notebook, python, jupyterlab, data science
+title: Run JupyterLab
+toc_min: 1
+toc_max: 2
+---
+
+## Overview
+
+This section walks you through how to run personal Jupyter Server with
+JupyterLab in a container. Containers ensure that your JupyterLab environment
+remains the same across different deployments, regardless of the operating
+system or setup. With a single Docker command, you can consistently run
+JuptyerLab on different machines.
+
+## Prerequisites
+
+You have installed the latest version of [Docker Desktop](../../../get-docker.md).
+
+## Run a JupyterLab container
+
+In a terminal, run the following command to run your JupyterLab container.
+
+```console
+$ docker run --rm -p 8888:8888 jupyter/base-notebook start-notebook.py --NotebookApp.token='my-token'
+```
+
+The following is a breakdown of the command:
+
+- The `docker run` command tells Docker to pull and run an image. In this case,
+the image is the `jupyter/base-notebook` image.
+- The `-p` option maps port `8888` on the host to port `8888` in the container.
+- `--rm` tells Docker to remove the image after it's stopped.
+- `start-notebook.py --NotebookApp.token='my-token'` is the command that the
+  server will run after starting. In this case, it's a script that sets the
+  access token for convenience. For more details about `start-notebook.py`, see
+  [Jupyter Server Options](https://jupyter-docker-stacks.readthedocs.io/en/latest/using/common.html#jupyter-server-options).
+
+To access the container, in a web browser navigate to [localhost:8888/lab?token=my-token](http://localhost:8888/lab?token=my-token).
+
+After accessing the URL, you should see the JupterLab interface. Since you specified `--rm` when running the container, the container and any data you create is deleted when the container stops.
+
+To stop the container, press `ctrl`+`c` in the terminal. If you see a prompt confirming shutdown in the terminal, specify `Y` and press `Enter`.
+
+## Save and access notebooks locally
+
+When you remove a container, all data in that container is deleted. To save
+notebooks or access notebooks on local machine, you can use a
+[bind mount](../../../storage/bind-mounts.md).
+
+### Run a JupterLab container with a bind mount
+
+Open a terminal, and change directory to a directory where you would like to save the notebook's data. Then, run the following command based on your operating system and shell.
+
+{{< tabs >}}
+{{< tab name="Windows (Command Prompt)" >}}
```

Comment:
This is one of the most common issues in the getting started guide now, even when specifically saying which shell to use. Based on the other comments, I'll change this to a volume, which should avoid the different syntax.

Reply:
Since we kept the bind mount for this initial example, I guess you'll want to add back the git bash / cmd tabs. Right now I only see PowerShell and Unix (which are identical)

Reply:
Oops! Yep.